zubhub_frontend/zubhub/src/App.js

```diff
@@ -86,423 +95,429 @@ function App(props) {
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={SearchResults} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/signup"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={Signup} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/login"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={Login} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/password-reset"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={PasswordReset} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/password-reset-confirm"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={PasswordResetConfirm} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/email-confirm"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={EmailConfirm} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/phone-confirm"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={PhoneConfirm} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/group-invite-confirm"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={GroupInviteConfirm} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
           />
 
           <Route
             path="/creators/:username/projects"
-            render={routeProps => (
+            element={
               <PageWrapper {...routeProps} {...props}>
                 <LazyImport LazyComponent={UserProjects} {...routeProps} {...props} />
               </PageWrapper>
-            )}
+            }
+          />
+
+          <Route
```

Comment:
why are we adding this?

Reply:
I had to register these back normally as they were Protected before. We already have a way to protect routes in the pagewrapper